ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel.ambr

```diff
@@ -33,29 +33,21 @@
                 (SELECT aggregation_target,
                         timestamp,
                         if(event = 'user signed up'
-                           AND (distinct_id IN
-                                  (SELECT distinct_id
-                                   FROM
-                                     (SELECT distinct_id, argMax(person_id, version) as person_id
-                                      FROM person_distinct_id2
+                           AND (aggregation_target IN
```

Comment:
@EDsCODE this snapshot update I'm not too sure of. It's a cohort subquery, but now it seems to skip person_distinct_id table completely? This looks correct to me, since aggregation target is the person_id, but could use another set of eyes here.

Reply:
It does seem like this new query is better because it filters directly on the aggregation_target (person_id_ rather than doing the distinct_id logic 🤔